app/src/main/res/layout/activity_launch_screen.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
<img width="264" alt="image" src="https://user-images.githubusercontent.com/32803402/215705810-357fa045-43aa-4198-a089-1a30da64aca5.png">
Could you improve margins on the screen? It doesn't match the design, button should be glued to the footer

Reply:
in themes.xml you could also update android:statusBarColor to white so it matches the screen color